brave/brave-core
# Migrate kHideButton preference to kShowButton preference

Resolves https://github.com/brave/brave-browser/issues/19695

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Install version of Brave without this fix
- Enable Brave Rewards
- Visit brave://settings/appearance
- Toggle on "Hide Brave Rewards icon" option
- Close Brave
- Upgrade to version with this fix, keeping old profile
- Visit brave://settings/appearance
- Verify that "Show Brave Rewards icon" is unchecked

Rerun same test, but keep "Hide Brave Rewards icon" off and ensure that "Show Brave Rewards icon" is checked after upgrade
